content/blog/2020-03-04-what-is-wrong-with-sklearn/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title:  "What's wrong with Scikit-Learn."
+author: "Guillaume Chevalier"
+date: '2020-01-03'
+slug: neat-machine-learning-pipelines
+type: post
+canonical: https://www.neuraxio.com/en/blog/scikit-learn/2020/01/03/what-is-wrong-with-scikit-learn.html
+categories:
+  - Software engineering
+  - Python
+tags:
+  - scikit learn
+  - machine learning
+  - deep learning
+  - clean code
+  - software architecture
+  - machine learning
+description: "Scikit-Learn’s “pipe and filter” design pattern is simply beautiful. But how to use it for Deep Learning, AutoML, and complex production-level pipelines?"
```

Comment:
I thought this would be for the [meta description](https://neilpatel.com/blog/meta-description-magic/) of the article in the search, which is usually close to max 160 chars. 

What should I replace this with? Maybe another phrase from the article? E.g.: 

> Scikit-Learn had its first release in 2007, which was a pre deep learning era. However, it’s one of the most known and adopted machine learning library, and is still growing.

Or: 

> TL;DR: How could things work to allow to do what’s in the above list with the Pipe and Filter design pattern / architectural style that is particular of Scikit-Learn?